<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMappingDialog</class>
 <widget class="QDialog" name="ColorMappingDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Color Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ui.qrc">
    <normaloff>:/icons/images/main.svg</normaloff>:/icons/images/main.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="11">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btEditColor">
       <property name="toolTip">
        <string>Change selected color</string>
       </property>
       <property name="statusTip">
        <string>Change selected color</string>
       </property>
       <property name="text">
        <string>&amp;Edit</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/pencil.svg</normaloff>:/icons/images/pencil.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btAddColor">
       <property name="toolTip">
        <string>Add a new color at the end of the list</string>
       </property>
       <property name="statusTip">
        <string>Add a new color at the end of the list</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/list-add.svg</normaloff>:/icons/images/list-add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btRemoveColors">
       <property name="toolTip">
        <string>Remove selected colors from the list</string>
       </property>
       <property name="statusTip">
        <string>Remove selected colors from the list</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/list-remove.svg</normaloff>:/icons/images/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btLoadColorList">
       <property name="toolTip">
        <string>Open a color list</string>
       </property>
       <property name="statusTip">
        <string>Open a color list</string>
       </property>
       <property name="text">
        <string>&amp;Load</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/document-open.svg</normaloff>:/icons/images/document-open.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btSaveColorList">
       <property name="toolTip">
        <string>Save colors list to a file</string>
       </property>
       <property name="statusTip">
        <string>Save colors list to a file</string>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/document-save-as.svg</normaloff>:/icons/images/document-save-as.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Available columns</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QListWidget" name="lstUsedColumns">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Available colors</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QListWidget" name="lstColumns">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="8">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btUseSelectedColumns">
       <property name="toolTip">
        <string>Use selected columns</string>
       </property>
       <property name="statusTip">
        <string>Use selected columns</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-next.svg</normaloff>:/icons/images/go-next.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btRemoveSelectedColumns">
       <property name="toolTip">
        <string>Remove selected columns</string>
       </property>
       <property name="statusTip">
        <string>Remove selected columns</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-previous.svg</normaloff>:/icons/images/go-previous.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="7">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btAutoAssignColors">
       <property name="toolTip">
        <string>Assign colors automatically</string>
       </property>
       <property name="statusTip">
        <string>Assign colors automatically</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-previous.svg</normaloff>:/icons/images/go-previous.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btRemoveSelectedColumnsColors">
       <property name="toolTip">
        <string>Remove colors from selected columns</string>
       </property>
       <property name="statusTip">
        <string>Remove colors from selected columns</string>
       </property>
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/close.svg</normaloff>:/icons/images/close.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="10">
    <widget class="QComboBox" name="cbColorMap">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="9" colspan="2">
    <widget class="QListWidget" name="lstColors">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::CopyAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QToolButton" name="btGenerateColors">
     <property name="toolTip">
      <string>Clear the color list and fill it with randomly generated colors</string>
     </property>
     <property name="statusTip">
      <string>Clear the color list and fill it with randomly generated colors</string>
     </property>
     <property name="text">
      <string>&amp;Generate colors</string>
     </property>
     <property name="icon">
      <iconset resource="ui.qrc">
       <normaloff>:/icons/images/magic.svg</normaloff>:/icons/images/magic.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorMappingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorMappingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
